src/course-about/CourseAboutPage.scss

```diff
@@ -0,0 +1,23 @@
+@import "course-intro/course-media/CourseMedia";
+
+.course-about-intro {
+  .pgn__card {
+    min-height: var(--catalog-course-about-page-intro-card-min-height, 15.75rem);
+
+    .pgn__card-header .pgn__card-header-content {
+      flex-direction: column-reverse;
```

Comment:
I'm perfectly happy with this as a solution to the "we want the card subtitle to be above the title" problem, but I'd like to confirm that the reordering is an actual requirement.

Reply:
I prefer the paragon defaults fwiw